alienwarearena_helper.user.js

```diff
@@ -876,13 +921,16 @@ Sorting from fresh ones to old ones.">Vote for newly uploaded ${sectionType}${(s
 				await visitNews();
 			} else if ($(this).data('awah-quest') === 'social') {
 				await shareSocial();
+			} else if ($(this).data('awah-quest') === 'replies') {
+				await postReplies();
 			// Non automatic stuff
 			} else if ($(this).data('awah-quest') === 'avatar') {
 				document.location.href = '/account/personalization';
 			} else if ($(this).data('awah-quest') === 'video') {
 				document.location.href = '/ucf/Video/new?boardId=464';
 			} else if ($(this).data('awah-quest') === 'forum') {
-				document.location.href = '/forums/board/113/awa-on-topic';
+				let dailyThreadId = await getDailyThread();
```

Comment:
What if there is no daily thread yet?
Or... there is, but its for yesterday quest?

Reply:
Right, we could catch if none found. Any idea about too old threads? There's only a date for the last post.

Reply:
Just check that filename has word "converse" in it. I doubt the same quest will be for two days in a row, so this approach can only fail in very rare case of someone bumping very old thread while the new thread is still no there.

Reply:
You've missed that this line is directing the user to the daily help thread if we don't know what to do.

We could probably also visit linked URL in the daily thread if the quest is to visit a link. If we're opening the thread anyway the date should be available.

Reply:
I think we should somehow be sure, that getDailyThread() returns only relevant todays thread. With that we can simply choose where to lead user by if(). To returned daily thread or to forum where user can see by him(her)self for clues.
P.S. Im really sorry I haven't much time for past and this week to actually code something.